src/System.Private.CoreLib/shared/System/Version.cs

```diff
@@ -306,7 +309,7 @@ public static Version Parse(string input)
         public static Version Parse(ReadOnlySpan<char> input) =>
             ParseVersion(input, throwOnFailure: true)!;
 
-        public static bool TryParse(string? input, out Version? result) // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
+        public static bool TryParse(string? input, [NotNullWhen(true)] out Version? result)
```

Comment:
The other `TryParse` override is missing this attribute in its out parameter.

Reply:
Good eye.  Fixed.